daily-report-dashboard/flask-frontend/templates/proxy_filings_summary.html

```
@@ -0,0 +1,94 @@
+                            <div id="prospectusByIndustry-{{ industry_index }}" class="content">
+                                <ul>
+                                    {% for filing in filings[10:] %}
+                                        <li>
+                                            <strong>
+                                                <a href="{{ url_for('routes.view_filing', filing_id=filing['filing_id']) }}">
+                                                    {{ filing['company_name'] }}
+                                                </a>
+                                            </strong>
+                                            <!-- Accordion for Text Sections -->
+                                            <button class="accordion" onclick="toggleContent('industryFiler-{{ industry_index }}-{{ loop.index }}')">Text Sections</button>
+                                            <ul id="industryFiler-{{ industry_index }}-{{ loop.index }}" class="content">
```

HTMLHint (1.5.0)

[error] The id value [ industryFiler-{{ industry_index }}-{{ loop.index }} ] must be unique.

(id-unique)